### ISSUE TITLE
Networking config file

### ISSUE THREAD

**Issue by hugusmaximus** [hermit-os/hermit-rs#515]
I wonder if it wold be possible to have some compile-time config file (not source code) so some critical parameters of networking (static IP address, DNS servers, dhcp on/off,...) could be setup there before compiling and without having to change source code.

**Comment by stlankes** [comment]
Yeah, we have to describe it better. It exists sone environment variables to define the static network configuration. See https://github.com/hermit-os/kernel/blob/main/src/executor/device.rs#L95

**Comment by hugusmaximus** [comment]
Sure, this is what I'm doing  (and also disabling the dhcpv4 feature) but just wondered if this could be in a separated config file... It's fine, I can go ahead by modifying those variables you mention.

**Comment by stlankes** [comment]
Yes, we should do it. I will talk @mkroening about it.
